release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2582,3 +2582,107 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../upgrading/automated_upgrades.adoc#install-config-upgrading-automated-upgrades[Performing
 Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-11-129]]
+=== RHBA-2019:1753 - {product-title} 3.11.129 Bug Fix Update
+
+Issued: 2019-07-23
+
+{product-title} release 3.11.129 is now available. The list of packages and
+bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2019:1753[RHBA-2019:1753] advisory.
+The container images included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2019:1754[RHBA-2019:1754] advisory.
+
+[[ocp-3-11-129-bug-fixes]]
+==== Bug Fixes
+
+* In OpenShift on Azure environments, conditional arguments were missing that
+would result in incorrect kubelet node names in certain cases. The conditionals
+to set `nodeName` in `node-config` were added, and now kubelet names can be set
+as required. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1656983[*BZ#1656983*])
+
+* Health check playbooks would assume `Curator` was a `deploymentconfig` instead
+of a `cronjob`, and would fail the check because the resource type had changed.
+Now, the health check playbook properly evaluates for a `cronjob` instead of a
+`deploymentconfig`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1676720[*BZ#1676720*])
+
+* Some {product-title} installations would fail because the `selinux`
+check was occurring in the `openshift_node` role instead of the `init` role.
+Now, the `selinux` check occurs earlier in the installation process and is
+completed successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1710020[*BZ#1710020*])
+
+* Access to the `ElasticSearch` root URL was denied from a project's pod in
+{product-title} 3.11 instances that had been upgraded from version
+3.10. This was due to strenuous permissions that denied non-administrative users
+access to the root endpoints. Now, permissions have been changed so that all
+users are able to access the root endpoints. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1710868[*BZ#1710868*])
+
+* `ElasticSearch` metrics were unavailable in the `Prometheus` role. Now, the
+`Prometheus` role has been enabled access to monitor all `ElasticSearch` indices.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1712423[*BZ#1712423*])
+
+* `ImageStreams` would fail if not using a hosted managed registry due to an unset
+`referencePolicy` field. Now, the dictionary has been changed to read and modify
+the `referencePolicy` as needed, and `ImageStreams` can be used without a hosted
+managed registry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1712496[*BZ#1712496*])
+
+* The `templateinstance` controller did not properly manage cluster level objects
+in it's create path, and as a result failed to create projects specified in
```

Comment:
in it’s -> in its

Reply:
Yep, that one slipped through. Thanks, fixed!